<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View</class>
 <widget class="QMainWindow" name="View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1859</width>
    <height>1063</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="s21::OpenGLWidget" name="widgetGl">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>1061</width>
      <height>1061</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>40</y>
      <width>563</width>
      <height>631</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="4" column="0">
      <widget class="QPushButton" name="load_settings">
       <property name="text">
        <string>Load settings</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="start_record">
       <property name="text">
        <string>Start record</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="import_file">
         <property name="text">
          <string>Import file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Information about the loaded model:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_file">
         <property name="text">
          <string>File name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_number_of_vertex">
         <property name="text">
          <string>Number of vertexs: 0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_number_of_facets">
         <property name="text">
          <string>Number of facets: 0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="save_image">
       <property name="text">
        <string>Save as image</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="save_settings">
       <property name="text">
        <string>Save settings</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QTabWidget" name="tabWidget">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="movable">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="general">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <attribute name="title">
         <string>General</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="5" column="0">
          <widget class="QLabel" name="label_move_z_2">
           <property name="text">
            <string>z:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_move_y_2">
           <property name="text">
            <string>y:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_move_x_2">
           <property name="text">
            <string>x:</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QSlider" name="move_y">
           <property name="minimum">
            <number>-100</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QSlider" name="move_x">
           <property name="minimum">
            <number>-100</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLabel" name="move">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Move </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignHCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSlider" name="rotate_x">
           <property name="minimum">
            <number>-720</number>
           </property>
           <property name="maximum">
            <number>720</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="label_move_z">
           <property name="text">
            <string>z:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QSlider" name="rotate_z">
           <property name="minimum">
            <number>-720</number>
           </property>
           <property name="maximum">
            <number>720</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QGroupBox" name="type_projection">
           <property name="minimumSize">
            <size>
             <width>261</width>
             <height>61</height>
            </size>
           </property>
           <property name="title">
            <string>Type projection</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignHCenter</set>
           </property>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>351</width>
              <height>25</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QRadioButton" name="radio_projection_orthogonal">
               <property name="text">
                <string>orthogonal</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radio_projection_central">
               <property name="text">
                <string>central</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_move_x">
           <property name="text">
            <string>x:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="choose_background_color">
           <property name="text">
            <string>Choose background color</string>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QSlider" name="move_z">
           <property name="minimum">
            <number>-1000</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="label_scale">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Scale</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignHCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Rotate</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignHCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSlider" name="rotate_y">
           <property name="minimum">
            <number>-720</number>
           </property>
           <property name="maximum">
            <number>720</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_move_y">
           <property name="text">
            <string>y:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QSlider" name="scale">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>300</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="lines_setup">
        <attribute name="title">
         <string>Lines</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="2" column="1">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>thickness</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignHCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QSlider" name="lines_thickness">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QGroupBox" name="type_lines">
           <property name="minimumSize">
            <size>
             <width>221</width>
             <height>61</height>
            </size>
           </property>
           <property name="title">
            <string>Type:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignHCenter</set>
           </property>
           <widget class="QWidget" name="layoutWidget_3">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>371</width>
              <height>25</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QRadioButton" name="radio_lines_solid">
               <property name="text">
                <string>solid</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radio_lines_dashed">
               <property name="text">
                <string>dashed</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QPushButton" name="choose_lines_color">
           <property name="text">
            <string>Choose lines color</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="vertexes_setup">
        <attribute name="title">
         <string>Vertexes</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="3" column="0" colspan="2">
          <widget class="QSlider" name="vertexes_size">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>50</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QPushButton" name="choose_vertex_color">
           <property name="text">
            <string>Choose vetexes color</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QGroupBox" name="display_method">
           <property name="minimumSize">
            <size>
             <width>211</width>
             <height>61</height>
            </size>
           </property>
           <property name="title">
            <string>Display method</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignHCenter</set>
           </property>
           <widget class="QWidget" name="layoutWidget_4">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>371</width>
              <height>25</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QRadioButton" name="radio_display_method_none">
               <property name="text">
                <string>none</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radio_display_method_circle">
               <property name="text">
                <string>circle</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radio_display_method_square">
               <property name="text">
                <string>square</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Size vertexes</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignHCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>s21::OpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">OpenGLWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
